specs.bzl

```diff
@@ -32,7 +32,7 @@ def _maven_repository(url, user = None, password = None):
         credentials = {"user": user, "password": password}
         return {"repo_url": url, "credentials": credentials}
 
-def _maven_artifact(group, artifact, version, packaging = None, classifier = None, override_license_types = None, exclusions = None, neverlink = None, testonly = None):
+def _maven_artifact(group, artifact, version = None, packaging = None, classifier = None, override_license_types = None, exclusions = None, neverlink = None, testonly = None):
```

Comment:
Version cannot be done, since this is used in maven_install and directly in coursier, and we want to enforce that the version is specified.

Reply:
Reverted.